ext/bin/config-puppet-test-ref

```diff
@@ -35,9 +35,9 @@ case "$#" in
         exit 2
 esac
 
-bundle install --without acceptance --path vendor/bundle
+bundle install --without acceptance --path vendor/bundle --retry=10
 bundle update
-bundle info puppet
+bundle info puppet || true  # Not all versions appear to have info
```

Comment:
I think this may be an artifact of Puppet not being installed properly. Can we give this a try without `|| true` now that it is working?